scripts/check-pvm-graph

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env fish
+
+if not which neo4j-admin
+	echo "Cannot find neo4j-admin in your current PATH ($PATH)"
+	exit 1
+end
+
+set dehydrated_db_dir $argv[1]
+if [ "$dehydrated_db_dir" = "" ]
+	echo "Usage:  run-neo4j <dehydrated DB directory>"
+	exit 1
+else if not [ -d "$dehydrated_db_dir" ]
+	echo "$dehydrated_db_dir is not a directory"
+	exit 1
+end
+
+set tmpdir (mktemp -d /tmp/neo4j.XXXXXX)
+
+echo "Hydrating $dehydrated_db_dir into $tmpdir"
+echo "Configuring temporary Neo4J instance..."
+
+
+#
+# Find an unused TCP port.
+#
+# This is racy, as we don't reserve the port, but it's better than
+# always using the same port...
+#
+function unused_port
+	set used_ports "$tmpdir/ports"
+	netstat -an \
+		| awk '/\([0-9]\+\.\)/{print $4}' \
+		| sed 's/[0-9\.]*\.//' \
+		| sort \
+		| uniq \
+		> $used_ports
+
+	while true
+		set port (random 1024 65000)
+		if grep $port $used_ports > /dev/null
+			sleep 1
+		else
+			echo $port
+			break
+		end
+	end
+end
+
+# Choose ports for Bolt and HTTP:
+set bolt_port (unused_port)
+set http_port (unused_port)
+
+#
+# Set up a Neo4j home directory structure:
+#
+mkdir $tmpdir/etc $tmpdir/neo4j
+
+set -x NEO4J_HOME $tmpdir/neo4j
+
+echo "\
+dbms.active_database=graph.db
+
+dbms.directories.data=./data
+dbms.directories.plugins=/usr/local/neo4j/lib/plugins
+dbms.directories.lib=/usr/local/neo4j/lib
+
+dbms.connector.bolt.enabled=true
+dbms.connector.bolt.listen_address=:$bolt_port
+
+dbms.connector.http.enabled=true
+dbms.connector.http.listen_address=:$http_port
+
+dbms.connector.https.enabled=false
+
+dbms.jvm.additional=-XX:+UseG1GC
+dbms.jvm.additional=-XX:-OmitStackTraceInFastThrow
+dbms.jvm.additional=-XX:+AlwaysPreTouch
+dbms.jvm.additional=-XX:+UnlockExperimentalVMOptions
+dbms.jvm.additional=-XX:+TrustFinalNonStaticFields
+dbms.jvm.additional=-XX:+DisableExplicitGC
+dbms.jvm.additional=-Dunsupported.dbms.udc.source=tarball\
+" > $tmpdir/etc/neo4j.conf
+
+set neo4j $tmpdir/neo4j
+mkdir -p $neo4j/logs $neo4j/run
+
+
+#
+# Import data into offline graph DB directory
+#
+
+# TODO: once dehydrated DBs start shipping with names like `nodes_*.csv`,
+#       use a glob rather than hardcoding filenames
+
```

Comment:
Potentially worth implementing.

Reply:
I think I recall the format of the dehydrated DBs having changed... is that right? If so, might you be able to share a new DB with me?